python-sdk/tests_integration/sql/operators/test_cleanup.py

```diff
@@ -28,164 +31,123 @@
     for database in Database
 ]
 
+drop_table_statement = "DROP TABLE IF EXISTS {table_name}"
+
 
-@pytest.mark.integration
 @pytest.mark.parametrize(
-    "database_table_fixture",
-    SUPPORTED_DATABASES_OBJECTS_WITH_FILE,
-    indirect=True,
-    ids=SUPPORTED_DATABASES,
+    "temp_table",
+    [
+        Table(conn_id="sqlite_conn"),
+        Table(conn_id="snowflake_conn"),
+        Table(conn_id="bigquery"),
+    ],
+    ids=["sqlite", "snowflake", "bigquery"],
```

Comment:
Should we add Redshift, postgres and delta too?

Reply:
added all three